### PR TITLE
WOPI Server deployment and configuration docs on Kubernetes

### PR DESCRIPTION
**Pull request by SamuAlfageme**
Initial draft to deploy and configure a Reva + WOPI Server + Collabora Code environment in Kubernetes.

### CODE REVIEW

File: docs/content/en/docs/IOP/Deployment/Kubernetes/wopiserver.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+helm repo update
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "sciencemesh" chart repository
+...Successfully got an update from the "stable" chart repository
+Update Complete. ⎈ Happy Helming!⎈
+
+helm search repo stable/collabora-code
+NAME                 	CHART VERSION	APP VERSION	DESCRIPTION
+stable/collabora-code	1.0.6        	4.0.3.1    	A Helm chart for Collabora Office - CODE-Edition
```
Comment (glpatcern) [inline docs/content/en/docs/IOP/Deployment/Kubernetes/wopiserver.md:46]: As just discovered, `collabora-code` must be version 4.2+

Comment (SamuAlfageme) [reply docs/content/en/docs/IOP/Deployment/Kubernetes/wopiserver.md:46]: Yep, I included a note pointing at this a few lines below on https://github.com/sciencemesh/sciencemesh/pull/142/commits/23f8b0edf6f15d343128272be4a2d3944008d6b1.

**Comment by SamuAlfageme** [comment]
Needs a brief update after the latest changes on https://github.com/cs3org/charts/pull/17, will push them asap.

**Comment by SamuAlfageme** [comment]
@labkode @glpatcern @Daniel-WWU-IT, this one is ready to review - needs https://github.com/sciencemesh/charts/pull/10 as we use `sciencemesh/meshapps` for installing Collabora CODE in these docs.

**Comment by SamuAlfageme** [comment]
This https://github.com/cs3org/charts/pull/16 and https://github.com/sciencemesh/charts/pull/11 should be merged as well to provide the:

```yaml
extraEnv:
   - name: REVA_APPPROVIDER_IOPSECRET
     valueFrom:
       secretKeyRef:
         name: iop-wopiserver-secrets
         key: iopsecret
```

... behavior used in these docs

**Review by labkode — APPROVED** [review]

**Pull request merged**